glide.yaml

```diff
@@ -16,7 +16,7 @@ import:
   version: ^1.3.0
 # Version 3.3.11 has incompatibilities
 - package: github.com/uber-go/tally
-  version: 3.3.10
+  version: master
```

Comment:
Can you pin this to v3.4.0 please? (Need this PR to land to unblock internal monorepo upgrades.)